commit fb032b271bea8dc2cd0c772ffd8767a30fcef279
Author: Joseph Salisbury <joseph.salisbury@canonical.com>
Date:   Wed Aug 14 12:53:36 2013 -0400

    UBUNTU: SAUCE: (no-up) intel_ips: blacklist ASUSTek G60JX laptops
    
    upstreamed
    
    BugLink: http://bugs.launchpad.net/bugs/1210848
    
    On an ASUSTek G60JX laptop, the intel_ips driver spams the log with a warning message: "ME failed to update for more than 1s, likely hung".  This ME doesn't support the feature, so requesting it be blacklisted for now.
    
    Signed-off-by: Joseph Salisbury <joseph.salisbury@canonical.com>
    Acked-by: Jesse Barnes <jbarnes@virtuousgeek.org>
    Tested-by: Nick Jenkins <tech.crew.jenkins@gmail.com>
    
    Acked-by: Seth Forshee <seth.forshee@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/drivers/platform/x86/intel_ips.c b/drivers/platform/x86/intel_ips.c
index 55663b3..7cfd555 100644
--- a/drivers/platform/x86/intel_ips.c
+++ b/drivers/platform/x86/intel_ips.c
@@ -1501,6 +1501,14 @@ static const struct dmi_system_id ips_blacklist[] = {
 			DMI_MATCH(DMI_PRODUCT_NAME, "HP ProBook"),
 		},
 	},
+	{
+		.callback = ips_blacklist_callback,
+		.ident = "G60JX",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "ASUSTeK Computer Inc."),
+			DMI_MATCH(DMI_PRODUCT_NAME, "G60JX"),
+		},
+	},
 	{ }	/* terminating entry */
 };
 
